bot/exts/evergreen/wikipedia.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+import datetime
+import logging
+from typing import List, Optional
+
+from discord import Color, Embed, Message
+from discord.ext import commands
+
+from bot.constants import Wikipedia
+
+log = logging.getLogger(__name__)
+
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
+WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+
+
+class WikipediaCog(commands.Cog):
+    """Get info from wikipedia."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.http_session = bot.http_session
+
+    async def search_wikipedia(self, search_term: str) -> Optional[List[str]]:
+        """Search wikipedia and return the first page found."""
+        async with self.http_session.get(SEARCH_API.format(search_term=search_term)) as response:
+            data = await response.json()
+        page = []
+
+        search_results = data["query"]["search"]
+        if len(search_results) == 0:
+            return None
+
+        # we dont like "may refere to" pages.
```

Comment:
```suggestion
        # Ignore pages with "may refer to"
```
Sounds better stay neutral, in addition the comment should probably state why we ignore them. Personally I have no idea just reading the comment. Is there no information on those pages?

Reply:
Additionally, how do you inform the user that their search got them one of these pages? Say it's the only result?